src/main/java/org/primefaces/util/Constants.java

```diff
@@ -88,6 +88,8 @@
 
     public static final String TABLE_STATE = "primefaces.TABLE_STATE";
 
+    public static final String LIST_STATE = "primefaces.LIST_STATE";
+
```

Comment:
Maybe DATALIST_STATE instead as LIST_STATE might be a little too ambiguous?

Reply:
Good point. Renamed the constant.